internal/rbd/replicationcontrollerserver.go

```diff
@@ -643,6 +642,11 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 
 			return nil, status.Error(codes.Internal, err.Error())
 		}
+
+		// If we issued a resync, return a non-final error as image needs to be recreated
+		// locally. Caller retries till RBD syncs an initial version of the image to
+		// report its status in the resync request.
+		return nil, status.Error(codes.Unavailable, "awaiting initial resync due to split brain")
```

Comment:
This should be unavailable or aborted?

Reply:
I usually go by [this](https://github.com/grpc/grpc-go/blob/0257c8657362b76f24e7a8cfb61df48d4cb735d3/codes/codes.go#L107-L109) definition, hence Unavailable, as this can be retried than starting at a higher point in the orchestration. The other places are Aborted, as the image may really not exist and orchestration may need to start at a higher level.